app/run.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+# Run a test server
+from ashlar import app
+
+app.run(port=4000, debug=True)
```

Comment:
Cool. Make sure you set the host parameter in the vagrant file via ENV.fetch() so it can be overridden.